### PR TITLE
Join table visits

### PR DESCRIPTION
**Pull request by Theophileaseh**
**In this project, I:**
Created a table named vets with the following columns:

- id: integer (set it as autoincremented PRIMARY KEY)
- name: string
- age: integer
- date_of_graduation: date

Inserted the following data for vets:

- Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
- Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
- Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
- Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.

Inserted the following data for specialties:

- Vet William Tatcher is specialized in Pokemon.
- Vet Stephanie Mendez is specialized in Digimon and Pokemon.
- Vet Jack Harkness is specialized in Digimon.

Inserted the following data for visits:

- Agumon visited William Tatcher on May 24th, 2020.
- Agumon visited Stephanie Mendez on Jul 22th, 2020.
- Gabumon visited Jack Harkness on Feb 2nd, 2021.
- Pikachu visited Maisy Smith on Jan 5th, 2020.
- Pikachu visited Maisy Smith on Mar 8th, 2020.
- Pikachu visited Maisy Smith on May 14th, 2020.
- Devimon visited Stephanie Mendez on May 4th, 2021.
- Charmander visited Jack Harkness on Feb 24th, 2021.
- Plantmon visited Maisy Smith on Dec 21st, 2019.
- Plantmon visited William Tatcher on Aug 10th, 2020.
- Plantmon visited Maisy Smith on Apr 7th, 2021.
- Squirtle visited Stephanie Mendez on Sep 29th, 2019.
- Angemon visited Jack Harkness on Oct 3rd, 2020.
- Angemon visited Jack Harkness on Nov 4th, 2020.
- Boarmon visited Maisy Smith on Jan 24th, 2019.
- Boarmon visited Maisy Smith on May 15th, 2019.
- Boarmon visited Maisy Smith on Feb 27th, 2020.
- Boarmon visited Maisy Smith on Aug 3rd, 2020.
- Blossom visited Stephanie Mendez on May 24th, 2020.
- Blossom visited William Tatcher on Jan 11th, 2021.

Wrote queries to answer the following:

- Who was the last animal seen by William Tatcher?
- How many different animals did Stephanie Mendez see?
- List all vets and their specialties, including vets with no specialties.
- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
- What animal has the most visits to vets?
- Who was Maisy Smith's first visit?
- Details for most recent visit: animal information, vet information, and date of visit.
- How many visits were with a vet that did not specialize in that animal's species?
- What specialty should Maisy Smith consider getting? Look for the species she gets the most.

[Screenshot](https://github.com/Theophileaseh/Vet-clinic-database/blob/1b16b1da7de955fece17be22d1890239aee36445/sss.png), [Screenshot](https://github.com/Theophileaseh/Vet-clinic-database/blob/1b16b1da7de955fece17be22d1890239aee36445/sss1.png)

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -6,4 +6,6 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
 
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (5, 'Charmander', '02-08-2020', 0, True, 10.23), (6, 'Plantmon', '11-15-2021', 2, True, 5.7), (7, 'Squirtle', '04-02-1993', 3, False, 12.13), (8, 'Angemon', '06-12-2005', 1, True, 45), (9, 'Boarmon', '06-07-2005', 7, True, 20.4), (10, 'Blossom', '10-13-1998', 3, True, 17), (11, 'Ditto', '05-14-2022', 4, True, 22);
 
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34), ('Jennifer Orwell', 19), ('Bob', 45), ('Melody Pond', 77), ('Dean Winchester', 14), ('Jodie Whittaker', 38);
 
+INSERT INTO species (name) VALUES ('Pokemon'), ('Digimon');
```
Comment (sja-thedude) [inline data.sql]: - [ ] Kindy Insert the data given in project requirements for vets, specialities and visits.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Theophileaseh, This is @sja-thedude your code reviewer.

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good PR description and title
✔️ Correct git-flow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (andersonlebon) [inline data.sql]: - [ ] Kindly Address the issue started by the previous reviewer you need to insert data in the vets, specialties, and visits tables.

image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @Theophileaseh 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Theophileaseh) [reply data.sql]: Hello, what you are asking me to do is what was done in the previous project or pull request. It is not for this one. You can check the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md)

Comment (Theophileaseh) [reply data.sql]: Hello, can you please check the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md) for this pull request? There's no problem there.

Comment (Theophileaseh) [reply data.sql]: Hello, can you please check the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md) for this pull request? There's no problem there.

Comment (andersonlebon) [reply data.sql]: Hello @Theophileaseh

Comment (Theophileaseh) [reply data.sql]: hello

Comment (andersonlebon) [reply data.sql]: - [ ] kindly check well the requirements of this project. it is clear that you should insert data in some tables.

Comment (Theophileaseh) [reply data.sql]: hello @andersonlebon

Comment (Theophileaseh) [reply data.sql]: @andersonlebon all the data has been inserted, you can check

**Review by aimemalaika — APPROVED** [review]
Hi @Theophileaseh , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**